src/main/resources/schema.sql

```diff
@@ -11,17 +22,19 @@ CREATE TABLE BOOKABLE (
 );
 
 CREATE TABLE LOCATION (
-    LOCATION_ID   VARCHAR(36) NOT NULL,
-    LOCATION_NAME VARCHAR(50) NOT NULL,
-    LOCATION_TZ   VARCHAR(50) NOT NULL
+    LOCATION_ID        VARCHAR(36) NOT NULL,
```

Comment:
add primary key whilst we're here?

Reply:
Heh.  Was trying to keep the blast radius related directly to the story/task, but yeah.